src/components/LanguageSelector.jsx

```diff
@@ -41,7 +42,7 @@ class LanguageSelector extends Component {
       existingLanguages.push(languageOption);
       menuLanguages = menuLanguages.filter(option => option !== languageOption);
     });
-    
+
```

Comment:
So, this is [the rule ](http://eslint.org/docs/rules/array-callback-return). Where is the function actually used? Could we extract it to its own name method?